sa/model.go

```diff
@@ -41,7 +41,9 @@ type regModel struct {
 
 // challModel is the description of a core.Challenge in the database
 //
-// The Validation field is a stub; the column is only there for backward compatibility.
+// XXX(#983): The TLS field is a stub; the column is only there for backward
```

Comment:
@jmhodges I'm OK dropping it in this PR.  Does that seem OK with you?  I thought folks might not want to throw away that data right away.